openc3/lib/openc3/core_ext/string.rb

```diff
@@ -389,4 +389,20 @@ def to_utf8!
     end
   end
 
-end # class String
+  def comment_erb
+    # Split with -1 to avoid stripping empty strings
+    output = self.split("\n", -1).collect! do |line|
+      # If we have a commented out line that starts with #
+      # but not followed by % (allows for disabling ERB comments),
+      # which contains an ERB statement (<% ...)
+      # then comment out the ERB statement (<%# ...).
+      # We explicitly don't comment out trailing ERB statements
+      # as that is not typical and is difficult to regex
+      if line.strip() =~ /^#[^%]*<%/
```

Comment:
You could also just add \s* to the regex instead of strip.   I wonder which is heavier